### ISSUE TITLE
Deliver app to Google Play

### ISSUE THREAD

**Issue by Ambrosiani** [NordicMuseum/Nordic-Museum-Audio-Guide#33]
creating a new issue for preparing the android version. Near Me functionality is skipped. See #17 for details on android/iOS considerations.

**Comment by Ambrosiani** [comment]
@RubenSandwich I'll put the android issues here. So, the current app in Google Play has the bundle identifier `com.nordic_museum_audio_guide_android`, while the ios app has SE.nordiskamuseet.audioguide and the v2 android app has `com.nordicmuseumaudioguide`. I tried changing the bundle identifier in the following two locations but then the build failed ("error: cannot find symbol variable BuildConfig") so I guess there are additional places that need changing:

https://github.com/NordicMuseum/Nordic-Museum-Audio-Guide/blob/3d4208e11a056220464a13e36d49adc3fdbd573d/v2/android/app/build.gradle#L131

https://github.com/NordicMuseum/Nordic-Museum-Audio-Guide/blob/3d4208e11a056220464a13e36d49adc3fdbd573d/v2/android/app/src/main/AndroidManifest.xml#L2

is there a script to change the bundle identifier which will correctly move/rename everything?

**Comment by RubenSandwich** [comment]
@Ambrosiani Not that I know of, but I'll look around the code latter today. Do you have Android Studio installed? If a tool exists for this purpose it is likely in Android Studio.

**Comment by Ambrosiani** [comment]
@RubenSandwich yup, seems like it's just renaming in a bunch of places: https://stackoverflow.com/questions/37389905/change-package-name-for-android-in-react-native

**Comment by Ambrosiani** [comment]
we need to use android APK Expansion files (or solve audio file loading in some other way?): https://developer.android.com/google/play/expansion-files
